File: Problem Set 6 (Python)/cash/cash.py
from cs50 import get_float

counter = 0
while True:
    change = get_float("Change owed: ")
    if change > 0:
        break

change = round(change*100,2)

while True:
    if change-25 >= 0:
        change = change - 25
        counter = counter + 1

    else:
        break

while True:
    if change-10 >= 0:
        change = change - 10
        counter = counter + 1

    else:
        break

while True:
    if change-5 >= 0:
        change = change - 5
        counter = counter + 1

    else:
        break

while True:
    if change-1 >= 0:
        change = change - 1
        counter = counter + 1

    else:
        break

print(counter)

File: Problem Set 1 (Objective-C)/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)

{

    float change;
    int counter = 0;

    //Get input
    do
    {
     change = get_float("Change owed: ");
    }
    while (change < 0);
    int cents = round(change * 100);

    //Check how many quarters
    do
    {
        if(cents-25 >= 0)
        {
            cents = cents - 25;
            counter = counter + 1;
        }
    }
    while(cents-25 >= 0);

    //Check how many dimes
    do
    {
        if(cents-10 >= 0)
        {
            cents = cents - 10;
            counter = counter + 1;
        }
    }
    while (cents - 10 >= 0);

    //Check how many nickels
    do
    {
        if(cents-5 >= 0)
        {
            cents = cents - 5;
            counter = counter + 1;
        }
    }
    while(cents-5 >= 0);

    //check how many pennies
    do
    {
        if(cents-1 >= 0)
        {
            cents = cents - 1;
            counter = counter + 1;
        }
    }
    while(cents-1 >= 0);

    //Print total count
    printf("%i\n", counter);

}